app/workers/alert_mailer_worker.rb

```diff
@@ -3,14 +3,14 @@
 class AlertMailerWorker
   include Sidekiq::Worker
 
-  def self.schedule_email(alert:, user:)
-    perform_async(alert.id, user.id)
+  def self.schedule_email(alert)
+    perform_async(alert.id, alert.target_user.id)
```

Comment:
The way sidekiq works is that it receives very simple things that lack state, because it stores them in Redis. So typically it should only get record IDs and not full objects. That's all fine here.

In this case though, we only need one object id — for the alert — to do all the work we need to do. We removed a call to `alert.email_target_user`, so we should just fetch that alert and call `email_target_user` on it. No need to give the worker any knowledge about the target user or how the method works.